development_workflow.md

```diff
@@ -0,0 +1,69 @@
+# Development / Contribution Guidelines
+
+## General Development Workflow in the Lab
+
+
+
+### Forking and Branching
+
+The lab's general workflow for contributing to a project involves:
```

Comment:
Yes, we have to add the solutions to splitting PRs up that we've discussed. We should add that to the section below on PRs. This section is meant to be a quick overview.